src/commands/serviceConnector/createServiceConnector.ts

```diff
@@ -3,25 +3,28 @@
 *  Licensed under the MIT License. See License.md in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
 
-import { createLinker } from "@microsoft/vscode-azext-serviceconnector";
+import { ServiceConnectorGroupTreeItem, createLinker } from "@microsoft/vscode-azext-serviceconnector";
 import { IActionContext } from "@microsoft/vscode-azext-utils";
 import { localize } from "../../localize";
 import { SiteTreeItem } from "../../tree/SiteTreeItem";
 import { createActivityContext } from "../../utils/activityUtils";
 import { pickWebApp } from "../../utils/pickWebApp";
 
-export async function createServiceConnector(context: IActionContext, item?: SiteTreeItem): Promise<void> {
+export async function createServiceConnector(context: IActionContext, item?: SiteTreeItem | ServiceConnectorGroupTreeItem): Promise<void> {
     item ??= await pickWebApp(context);
+    if (item instanceof SiteTreeItem) {
+        item = <ServiceConnectorGroupTreeItem>item.parent;
```

Comment:
This seems backwards to me.

If it was a SiteTreeItem, would its parent be the subscription node?
But the ServiceConnectorGroupTreeItem's parent would be the SiteTreeItem, right?

Reply:
That is true it wasn't causing any problems for some reason since the type of item when using the command palette was not a SiteTreeItem so it was never calling that line. 

I'll flip it but with that implementation I don't thing resourceId will be needed anymore.

Reply:
Yeah, I believe that you're right. I know I just approved it, but we could probably disregard the `resourceId` added in this [PR](https://github.com/microsoft/vscode-azuretools/pull/1539#pullrequestreview-1559486372)